devlib/__init__.py

```diff
@@ -56,7 +56,7 @@
 from devlib.utils.version import get_commit as __get_commit
 
 
-__version__ = '1.0.0'
+__version__ = '1.0.dev1'
```

Comment:
Please expand commit message to state the reason for this change.

Reply:
Updated.